src/plugins/intel_cpu/src/dnnl_postops_composer.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief Convinience wrapper class for handling OneDNN attributes & post ops.
+ * @file dnnl_postops_composer.h
+ */
+#pragma once
+
+#include <dnnl_types.h>
+
+#include <string>
+
+#include "cpu_memory.h"
+#include "memory_desc/cpu_memory_desc.h"
+#include "memory_desc/dnnl_blocked_memory_desc.h"
+#include "onednn/dnnl.h"
+
+namespace ov {
+namespace intel_cpu {
+
+class Node;
+
+// so far the API only support per-Tensor or per-OC
+class DnnlPostOpsComposer {
+public:
+    DnnlPostOpsComposer(ov::intel_cpu::Node* node,
+                        dnnl::primitive_attr& attr,
+                        dnnl::post_ops& ops,
```

Comment:
Yes, it is a compromised design, ideally this class should be the one and only way to compose an `primitive_attr`, and `post_ops` should be hidden to the caller, but that requires changing the legacy API `appendPostOps` and their implementation, more over, if we do so, then some node with our own jit implementation (like MVN) may have difficulty to understand the optimizations done by `DnnlPostOpsComposer`, for example, they don't understand binary post ops  so far, thus we designed `DnnlPostOpsComposer` in assumption that it's not the only one that writes into the post ops.

Reply:
I am not sure we will use DnnlPostOpsComposer for CPU JIT kernels in its current state, given we would prefer to have these kernels independent from oneDNN at all.
I see it like we will need to have base class PostOpsComposer with two inheritance: DNNLPostOpsComposer and CPUPostOpsComposer. General math optimizations will be in base class while append specifics in derived ones. Inheritance is not mandatory here: need to check on common interface possibility/necessity. 
Anyway do I understand correctly that since we are using DNNLPostOpsComposer right now is used only in DNNL based ops and it is enough to pass attrs only?

Reply:
Yes, this class is used for Conv/Deconv/Fullyconnect/Matmul only. maybe next time we can refine the code further by passing attrs only.